deploy/osd-custom-domains/01-dedicated-admins-cluster.ClusterRole.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-custom-domains-dedicated-admin-cluster
```

```suggestion
  labels:
    managed.openshift.io/aggregate-to-dedicated-admins: "cluster"
  name: osd-custom-domains-dedicated-admin-cluster
```

And remove the ClusterRoleBinding.  The label will aggregate to the `dedicated-admins-cluster` ClusterRole and it'll Just Work.